src/sql/base/browser/ui/editableDropdown/dropdownTree.ts

```diff
@@ -43,44 +43,44 @@ export class DropdownRenderer implements tree.IRenderer {
 		return { label, row };
 	}
 
-	public renderElement(tree: tree.ITree, element: Resource, templateId: string, templateData: Template): void {
+	public renderElement(tree: any, element: Resource, templateId: string, templateData: Template): void {
 		templateData.label.innerText = element.value;
 		templateData.row.title = element.value;
 	}
 
-	public disposeTemplate(tree: tree.ITree, templateId: string, templateData: Template): void {
+	public disposeTemplate(tree: any, templateId: string, templateData: Template): void {
```

Comment:
why are all of these becoming `any`? Is there a more precise type we can use? `any` makes it hard to maintain

Reply:
Its mainly since those aren't being used by the function but are necessary parts of the function signiture. I could make them `tree.ITree | undefined` to be cleaner

Reply:
That’s probably better just in case anyone comes along later who wants to use them